scripts/docs-versions.sh

```diff
@@ -8,8 +8,7 @@ awk '
   BEGIN {
     print "{"
     print "  \"versions\": ["
-    getline current_version < "src/VERSION"
-    printf "    {\"name\": \"" current_version "\", \"url\": \"/api/master/\", \"released\": false}"
+    printf "    {\"name\": \"master\", \"url\": \"/api/master/\", \"released\": false}"
```

Comment:
```suggestion
    printf "    {\"name\": \"nightly\", \"url\": \"/api/master/\", \"released\": false}"
```

Reply:
Agreed, that might even be more accurate. It's at https://crystal-lang.org/api/master/ though, not https://crystal-lang.org/api/nightly/
I wouldn't like to create a dicrepancy here.

Maybe we should consider to rename the URL... I don't think it's a huge deal, however.

Reply:
I could even easily argue away the "inconsistency" :) The URL is a technical detail stemming from a technical artifact of our development process, that the current nightly build and docs are based on whatever is currently in the git "master" branch. That's no reason for the technical detail to leak into the user facing description of what it is.

Reply:
But there is also a discrepancy between https://crystal-lang.org/api/master/ and https://github.com/crystal-lang/crystal/tree/master

Reply:
I'm fine with changing the label (for now) to latel :+1: